### ISSUE TITLE
Additional data for trained PP-OCRv3 model

### ISSUE THREAD

**Issue by MyraBaba** [PaddlePaddle/PaddleOCR#10087]
I want to improve the Chinese and English ultra-lightweight PP-OCRv3 model（16.2M） model which one using in the license plate detection. 

How I can use the already pre-trained model to utilize it to add my data to improve accuracy ?

transfer_learning ...
it would be hande-y for command line info also.

**Comment by livingbody** [comment]
The tutorial is as follows：

PaddleOCR/doc/doc_ch/finetune.md at release/2.6 · PaddlePaddle/PaddleOCR · GitHub
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/finetune.md

**Comment by shiyutang** [comment]
The above question has answered the question, welcome to reopen the question if there are any following issues. And we are now holding a contribution to Paddleseg and Paddle OCR activity, you are welcome to join:[#10223](https://github.com/PaddlePaddle/PaddleOCR/issues/10223)
